backend/README.md

```diff
@@ -27,14 +27,19 @@ the regular base application.
 The project requires a couple of environment variables to run, if you want to develop on this codebase.
 Setting values for these variables can be done with a method to your own liking.
 
-| Variable          | Description                                                    |
-|-------------------|----------------------------------------------------------------|
-| DB_HOST           | Url of where the database is located                           |
-| POSTGRES_USER     | Name of the user, needed to login to the postgres database     |
-| POSTGRES_PASSWORD | Password of the user, needed to login to the postgres database |
-| POSTGRES_HOST     | IP adress of the postgres database                             |
-| POSTGRES_DB       | Name of the postgres database                                  |
-| API_HOST          | Location of the API root                                       |
+| Variable                               | Description                                                                                                                               |
+|----------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| POSTGRES_USER                          | Name of the user, needed to login to the postgres database                                                                                |
+| POSTGRES_PASSWORD                      | Password of the user, needed to login to the postgres database                                                                            |
+| POSTGRES_HOST                          | IP adress of the postgres database                                                                                                        |
+| POSTGRES_DB                            | Name of the postgres database                                                                                                             |
+| API_HOST                               | Location of the API root                                                                                                                  |
+| CLIENT_ID                              | [Client id](https://learn.microsoft.com/nl-nl/entra/identity-platform/v2-protocols)                                                       |
+| CLIENT_SECRET                          | Client's secret is your personal secret key for authentication                                                                            |
+| CODE_VERIFIER                          | Code verifier used to verify your personal code in the microsoft application                                                              |
+| JWT_SECRET_KEY                         | JWT secret key is used for security reasons for the microsoft application                                                                 |                       
```

Comment:
JWT secret key and what it's used for is no explanation of what the env variable must be

Reply:
I have no idea what it must do, what does the env variable do?

Reply:
JWT secret key is the key used to encode the JWT's and should be kept secret, because otherwise everyone can create "valid" JWT's for our application.

Reply:
It should just be a randomized string of 32 characters long

Reply:
The specifications for proper secret keys are defined in [RFC](https://www.rfc-editor.org/rfc/rfc4868#page-3)

Reply:
https://github.com/SELab-2/UGent-3/pull/357/commits/20edcfdc8693f103d8c2158114183a2585291164

Reply:
https://github.com/SELab-2/UGent-3/pull/357/commits/00064c497d5018a92980f1cfdac5dae515ba5f5f